packages/realm-server/worker.ts

```diff
@@ -85,7 +85,7 @@ let dist: URL = new URL(distURL);
 
 (async () => {
   let dbAdapter = new PgAdapter();
-  let queue = new PgQueue(dbAdapter);
+  let queue = new PgQueueRunner(dbAdapter, 'realm-worker'); // TODO: update worker ID with ecs task ID
```

Comment:
@habdelra If you can add this, we should be good to merge from my perspective

Reply:
done in https://github.com/cardstack/boxel/pull/1605/commits/244d9653d192270220e93a5711f4f9f6e451f679